redisinsight/ui/src/packages/redisearch/src/components/TableInfoResult/TableInfoResult.tsx

```diff
@@ -70,7 +70,7 @@ const TableInfoResult = React.memo((props: Props) => {
         <>
           <EuiText className="row" size="s" color="subdued">
             Indexing
-            <GroupBadge type={result?.index_definition?.key_type?.toLowerCase()} className="badge" />
+            <GroupBadge type={result?.index_definition?.data_type?.toLowerCase()} className="badge" />
```

Comment:
Hello
In this place we can't change `key_type` this field we receive from the RediSearch module command `FT.INFO <idx>` 
https://oss.redis.com/redisearch/Commands/#:~:text=5)%20index_definition%0A%206)-,1)%20key_type,-2)%20HASH%0A%20%20%20%20%203

Reply:
In my opinion, This needs to communicated to RedisSearch team and they may have plans to bring consistency on term usage in their Product backlogs. If you see `FT.CREATE` it uses `data_type`.

Reply:
I am unable to find visually any issues or test case failure.. if you don't mind can you help me by sharing what exactly is breaking with this change and how do you want this to be changed in your view, @zalenskiSofteq ?

Reply:
Thank you for your interest.

**First part:**
RediSearch is a plugin for our app. It isn't a part of app. Users able create a plugins and put them in to our app. More info about build and use plugins in this page: https://github.com/RedisInsight/RedisInsight/wiki/Plugin-Documentation/

RediSearch plugin visualizes two commands `FT.SEARCH` and `FT.INFO`

Simple way for tests changes  in this redisearch plugin I advise: 

1. Go to the local folder https://github.com/RedisInsight/RedisInsight/tree/main/redisinsight/ui/src/packages/redisearch
2. Run `yarn install`
3. Uncomment this [line](https://github.com/RedisInsight/RedisInsight/blob/bbc7c36405d2a6d2cbdbf2926bafb59c20864bfe/redisinsight/ui/src/packages/redisearch/src/main.tsx#L5) and this [line](https://github.com/RedisInsight/RedisInsight/blob/bbc7c36405d2a6d2cbdbf2926bafb59c20864bfe/redisinsight/ui/src/packages/redisearch/src/main.tsx#L22) for mock **command** and **result** for for `FT.INFO` command
4. Run `yarn start`
5. Open `localhost:1234` in the browser
6. Do any changes
_If you have any problems with this way please feel free to ask_

your change should break budge with **key_type**
![image](https://user-images.githubusercontent.com/63463140/149786916-3ddf333c-c0ed-47ad-b9cd-082105a16485.png)


**Second part:** 
According rename `key_type` to `data_type` field in the RediSearch module response for `FT.INFO`
If a new RediSearch version has `data_type` the previous versions will have steel `key_type` field. And for this plugin we have to maintenance both fields and `data_type` and `key_type` for oldest versions of RediSearch

Reply:
Curious to know, why there are API, plugin and app all in one repo? 🤔

Reply:
> Regarding the steps you provided, I am getting below error is there any external dependencies? What am I missing? 🤔

Try
1. Run `yarn build:statics` for Linux or Macos or `yarn build:statics:win` for Windows in the root folder
2. Again run `yarn start` in the redisearch folder

> This can be done getting RedisSearch version ahead of time, we already do that on any connection made to Redis we pull this info along with client list etc. We may have to keep both until last version which is sunset.

It is depends on configuration of connected database. In some cases we don't have version of module from Redis Enterprise or Redis Cloud

Reply:
> Curious to know, why there are API, plugin and app all in one repo?

1. API and plugins don't be used without our app
2. Common Eslint\Tsconfig rules
3. Plugins have to contain built styles from main app to be look like main app. 
4. Other multi-packages repos [Nest.js](https://github.com/nestjs/nest/tree/master/packages), [electron-builder](https://github.com/electron-userland/electron-builder/tree/master/packages), [babel](https://github.com/babel/babel/tree/main/packages)

Reply:
> It is depends on configuration of connected database. In some cases we don't have version of module from Redis Enterprise or Redis Cloud


Hmmm.. 🤔 how about directly using index position of the `index_definition` than `key_type` name?

Reply:
> your change should break budge with **key_type** 

![image](https://user-images.githubusercontent.com/63463140/149786916-3ddf333c-c0ed-47ad-b9cd-082105a16485.png)

Hey,
What could be the reason that I get same result as what you shared in this image with branch I created for PR or main branch (with incognito mode)?

Reply:
@zalenskiSofteq, screenshot I shared is how it really should work, correct?

Reply:
Yes, this screenshot is how really should work. 
Any other changes in the TableInfoResult.tsx component change anything in the localhost:1234?

Reply:
@zalenskiSofteq - pushed change to this specific issue. Now it should work. Can you review once again and let me what do you think of the approach.